vispy/gloo/glir.py

```diff
@@ -233,7 +233,7 @@ def convert_shaders(convert, shaders):
                 lines.append(line.rstrip())
             # Write
             if not has_version:
-                lines.insert(0, '#version 120\n#line 2\n')
```

Comment:
@izaid `git blame` says this line is yours -- why add the `#line 2` comment? Was that a leftover from debugging?